src/interfaces/ISablierV2ProxyTarget.sol

```diff
@@ -87,6 +87,21 @@ interface ISablierV2ProxyTarget {
     )
         external;
 
+    /// @notice Withdraws assets from streams to the recipient.
```

Comment:
Let's rewrite to:

> Withdraws assets to the recipient of each stream.

Reply:
I followed the format used in `ISablierV2Lockup`. 

https://github.com/sablier-labs/v2-core/blob/91887daff5872c7fa83fe3644b6f7582e9bffc0d/src/interfaces/ISablierV2Lockup.sol#L294

I suggest we keep like this for consistency. Or we should change both.

Reply:
"Formatting" is about white spaces, print widths, and other stylistic things. "Wording" is what we are talking about there, and wording does not have to be "consistent" (i.e., identical) when the context requires a different formulation.

"to the recipient of each stream" is much, much, much clearer than "from streams to the recipient" (the latter sounds like you're withdrawing to one account only).

Reply:
> Formatting" is about white spaces, print widths, and other stylistic things. 

lol😅  I was not referring to this. let's not debate words, but the context was "same style or structure of phrasing" or "wording format"

> much, much clearer than "from streams to the recipient" (the latter sounds like you're withdrawing to one account only).

It is clearer. It might be me but I have this thing about "general" consistency in everything, like maker's vocabular.

Anyway, I pushed a commit with the version suggested.

Reply:
Thanks!

> the context was "same style or structure of phrasing" or "wording format"

Ok, now I get it. This is not the typical way "format" is used, though.

>  like maker's vocabular.

I am staunchly against Maker's approach.